app/Actions/Albums/Top.php

```diff
@@ -60,7 +61,7 @@ public function get(): TopAlbums
 		$smartAlbums = $this->albumFactory
 			->getAllBuiltInSmartAlbums(false)
 			->map(
-				fn ($smartAlbum) => $this->albumAuthorisationProvider->isVisible($smartAlbum) ? $smartAlbum : null
+				fn ($smartAlbum) => Gate::check('see', $smartAlbum) ? $smartAlbum : null
```

Comment:
I don't really like the term `see` -- I would prefer to use `visible` instead, which seems to better convey what it's about? Or is it meant to be a verb?

Reply:
Consider this from the point of view of the User: "_Can current user see this smartAlbum?_"
Likewise, the property isEditable being switched from User point of view: "_Can curent User edit this?_"

Reply:
However one could also put the sentence the other way around which would allow to keep the old phrasing:

- Is this `model` `visible` by `user``
- Is this `model` `editable` by `user``

Both `AlbumAuthorizationProvider` and `PhotoAuthorizationProvider` contain long explanations in form of comments which define all the terms in a very precise way. Unless you do not want to rewrite all that definitions, comments and also all occurrences elsewhere, I would _strongly_ argue to keep the current wording.

One of the former problems was that Lychee used the same term for different things, but also different terms for the same thing. It was a lot of work to fix this problem in both directions: on the one hand to ensure that whenever a term is used it always means exactly the same no matter of the context; on the other hand to ensure that whenever the same thing is meant it is referred to by the same term always.

Introducing a new term for the same thing will soon lead to the previous unfortunate situation.

Reply:
> However one could also put the sentence the other way around which would allow to keep the old phrasing:
> 
> * Is this `model` `visible` by `user``
> * Is this `model` `editable` by `user``

Good point. Will fix.

Reply:
fixed.